testsuite/features/support/namespaces/kickstart.rb

```diff
@@ -66,6 +66,49 @@ def initialize(api_test)
     @test = api_test
   end
 
+  ##
+  # Creates a Kickstart tree (Distribution).
+  #
+  # Args:
+  #   distro:       The name of the kicktart tree (distribution) you want to create.
+  #   path:         Path to the base or root of the distribution.
+  #   label:        Label of the channel you want to associate with the disribution.
+  #   install:      Label for KickstartInstallType - options can be obtained using kickart.tree's listInstallTypes.
+  #                 Options can be: suse, sles15generic, sles12generic, sles11generic, sles10generic, rhel_9, rhel_8, rhel_7, rhel_6, generic_rpm, fedora18.
+  def create_distro(distro, path, label, install)
+    @test.call('kickstart.tree.create', sessionKey: @test.token, treeLabel: distro, basePath: path, channelLabel: label, installType: install)
+  end
+
+  ##
+  # Creates a Kickstart tree (Distribution), adding kernel options as parameters.
+  #
+  # Args:
+  #   distro:       The name of the kicktart tree (distribution) you want to create.
+  #   path:         Path to the base or root of the distribution.
+  #   label:        Label of the channel you want to associate with the disribution.
+  #   install:      Label for KickstartInstallType - options can be obtained using kickart.tree's listInstallTypes.
+  #                 Options can be: suse, sles15generic, sles12generic, sles11generic, sles10generic, rhel_9, rhel_8, rhel_7, rhel_6, generic_rpm, fedora18.
+  #   options:      Options to be passed to the kernel when booting for the installation.
+  #   post_options: Options to be passed to the kernel when booting for the installation.
```

Comment:
Shouldn`t there be a difference in the explanation here? Same for the other method below.

Reply:
it was taken directly from the documentation like that, I'm afraid :grimacing: 
![image](https://user-images.githubusercontent.com/1148556/226959382-2a2b995c-889f-4587-a7df-2ebbad2f47bc.png)
but open to ideas

Reply:
I see. It makes sense to just copy paste the official descriptions. Then I guess all good and the API documentation should be adjusted.